src/PermissionlessGate.sol

```diff
@@ -0,0 +1,100 @@
+// SPDX-FileCopyrightText: 2024 Lido <info@lido.fi>
+// SPDX-License-Identifier: GPL-3.0
+
+pragma solidity 0.8.24;
+
+import { ICSAccounting } from "./interfaces/ICSAccounting.sol";
+import { ICSModule, NodeOperatorManagementProperties } from "./interfaces/ICSModule.sol";
+import { IPermissionlessGate } from "./interfaces/IPermissionlessGate.sol";
+
+/// @title PermissionlessGate
+/// @notice Contract for adding new Node Operators with no any restrictions
+contract PermissionlessGate is IPermissionlessGate {
+    /// @dev Curve ID is the default bond curve ID from the accounting contract
+    ///      No need to set it explicitly
+    uint256 public immutable CURVE_ID;
+
+    ICSModule public immutable CSM;
+    ICSAccounting public immutable ACCOUNTING;
```

Comment:
I can't see why we need it as immutable. It is only used in constructor

Reply:
oh right